lib/absinthe/middleware.ex

```diff
@@ -310,6 +310,9 @@ defmodule Absinthe.Middleware do
       [{:ref, Absinthe.Type.BuiltIns.Introspection, _}] ->
         expanded
 
+      [{:ref, Absinthe.Phase.Schema.DeprecatedDirectiveFields, _}] ->
```

Comment:
Not really happy with this, but can be removed when these fields are no longer necessary.